client/components/deposits-status/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+
+const DepositsEnabled = ( props ) => {
+	const { depositsEnabled, iconSize } = props;
+	let className = 'account-details__info__green';
+	let description;
+	let icon = <Icon icon="yes-alt" size={ iconSize } />;
```

Comment:
The `iconSize` prop in all these components doesn't change - can it just be hardcoded instead? This way we "save" a prop